apps/balancer-tools/src/app/apr/(components)/KPI.tsx

```diff
@@ -2,7 +2,7 @@ import { InfoCircledIcon } from "@radix-ui/react-icons";
 import { Suspense } from "react";
 
 import { Spinner } from "#/components/Spinner";
-import { Tooltip } from "#/components/Tooltip";
+import { TooltipMobile } from "#/components/TooltipMobile";
```

Comment:
Do we need the name change? Couldn't this simply be a tooltip that works on mobile? 🤷